doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.rst

```diff
@@ -15,18 +15,26 @@ The entire code can be seen :codedir:`here in the moveit_pr2 github project<plan
 
 The launch file
 ^^^^^^^^^^^^^^^
-The entire launch file is `here
-<https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
+The entire launch file is `here <https://github.com/ros-planning/moveit_tutorials/tree/kinetic-devel/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
 on github. All the code in this tutorial can be run from the
 moveit_tutorials package that you have as part of your MoveIt! setup.
 
 Running the code
 ^^^^^^^^^^^^^^^^
 
-Roslaunch the launch file to run the code directly from moveit_tutorials::
+Make sure your python file is executable (chmod +x scripts/move_group_python_interface_tutorial.py).
+
+Either run the python code directly using Rosrun::
+
+ rosrun moveit_tutorials move_group_python_interface_tutorial.py
+
+Or roslaunch the launch file to run the code directly from moveit_tutorials::
```

Comment:
Can you remove the roslaunch file and instead make the instructions simply say A) run ``roslaunch pr2_moveit_config demo.launch`` then B) run ``rosrun moveit_tutorials move_group_python_interface_tutorial.py``? Similar to the tutorial I just modified for C++ [here](https://github.com/davetcoleman/moveit_tutorials/blob/73d51d377ed960c40980c28042f45cd4b630cfd2/doc/pr2_tutorials/planning/src/doc/move_group_interface_tutorial.rst#start-rviz-and-movegroup-node)